### PR TITLE
add tril op and  triu op

### PR DESCRIPTION
**Pull request by WuHaobo**
1. add new op  paddle.tensor.triu
image
2. add new op  paddle.tensor.tril
image
3. add the unittest and api test of the two new OPs

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_tril_triu_op.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+
+### NOTE: meaningful diagonal is [1 - min(H, W), max(H, W) -1]
+### test the diagonal just at the border, upper/lower the border, 
+###     negative/positive integer within range and a zero
+cases = {
+    'success': {
+        (2, 2, 3, 4, 5): [-100, -3, -1, 0, 2, 4, 100],  # normal shape
+        (10, 10, 1, 1): [-100, -1, 0, 1, 100],  # small size of matrix
+        # (1024, 1024): [-1023, -857, 0, 505, 1023], # large size of matrix
```
Comment (heavengate) [inline python/paddle/fluid/tests/unittests/test_tril_triu_op.py]: uncomment or delete

**Comment by heavengate** [comment]
> No description provided.

请写一下PR comment~

**Review by heavengate — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_tril_triu_op.py (hunk 1 above)
Comment (WuHaobo) [reply python/paddle/fluid/tests/unittests/test_tril_triu_op.py]: Done

Comment (WuHaobo) [reply python/paddle/fluid/tests/unittests/test_tril_triu_op.py]: > uncomment or delete

Done

**Pull request merged**